docs/Azure_IoT_Integration_Guide/Concepts/LwM2M_mappings.md

```diff
@@ -110,22 +128,29 @@ Thus, for instance, to execute a factory reset on a device, you need to invoke a
        path: "3.0.5"
     }
 ```
+
+In **Azure IoT Central**, select a device from the **Devices** page and click the **Commands** tab. Click **Run** to run a necessary command.
+
+![Device twin READ](images/commands.png "Commands tab")
+
 ### EXECUTE - Communication flow
 
-Invoking a direct method from Azure IoT Hub and handling it by Coiote DM in the form of an EXECUTE operation passed to the device has the following flow:
+Invoking a direct method from Azure IoT Hub or Azure IoT Central and handling it by Coiote DM in the form of an EXECUTE operation passed to the device has the following flow:
```

Comment:
w Azure IoT Central to jest command. Wczesniej o tym wspominajsz, więc chyba dobrze być spójnym.

Reply:
tak, to ważne, dzięki, że zauważyłeś